### PR TITLE
Enable publishing to npm

### PR DESCRIPTION
**Pull request by kichalla**
@Eilon @DamianEdwards

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,35 @@
 {
-  "name": "ASP.NET",
-  "version": "0.0.0",
-  "devDependencies": {
-    "gulp": "3.8.11",
-    "gulp-rename": "1.2.2",
-    "gulp-uglify": "1.2.0"
-  }
+    "name": "jquery-validation-unobtrusive",
```
Comment (kichalla) [inline package.json]: For reference, you can use bower.json for most of the content here: https://github.com/aspnet/jquery-validation-unobtrusive/blob/master/bower.json

**Comment by Eilon** [comment]
This all looks fine to me, but deferring to @DamianEdwards for final code review.

**Comment by kichalla** [comment]
ping @DamianEdwards

**Comment by spboyer** [comment]
Closes #17 

:+1:

**Comment by DamianEdwards** [comment]
Looks good to me :shipit:

**Comment by kichalla** [comment]
41bf5b4c4d8bcf44dd810bf3bf96cc87f2c49315

**Pull request closed without merging**